docs/builder-single-focus-redesign/mockups/single-focus-immersive-v2.html

```
@@ -0,0 +1,137 @@
+<h2>Single-focus v2 — carousel moved to bottom, identity clustered with the sprite</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)